build/__nodeVersions.sh

```diff
@@ -9,4 +9,5 @@ NODE6_VERSION='6.17.1'
 NODE8_VERSION='8.17.0'
 NODE10_VERSION='10.19.0'
 NODE12_VERSION='12.16.1'
+NODE14_VERSION='14.0.0'
```

Comment:
Can you add tests for this? You can take a look at existing tests for version `12` to get an idea.